### PR TITLE
"Super" segment

### PR DESCRIPTION
**Pull request by christophe-david**
This PR allows to make any implemented flight part usable as segment in mission files.

A Takeoff super-segment is added, which is a combination of the 3 "atomic" takeoff segments (ground speed acceleration, rotation, end of takeoff).

### CODE REVIEW

File: src/fastoad/models/performances/mission/segments/tests/test_takeoff_segments.py [hunk 1]
```diff
@@ -119,15 +122,14 @@ def test_rotation(polar, polar_modifier):
     assert_allclose(last_point.time, 32.58, rtol=1e-2)
     assert_allclose(last_point.mass, 69995.0, rtol=1e-4)
     assert_allclose(last_point.ground_distance, 200.64, rtol=1e-3)
-    assert_allclose(last_point.alpha, 0.134, rtol=1e-2)
```
Comment (esnguyenvan) [inline src/fastoad/models/performances/mission/segments/tests/test_takeoff_segments.py:122]: I see this assertion was removed, I wonder for which reason?

**Review by esnguyenvan — APPROVED** [review]
Hello, thanks for your contribution. The super segment will be useful for both takeoff, landing and others (start-stop, takeoff with engine failure etc...).
Only have minor remarks from my side.

File: src/fastoad/models/performances/mission/segments/base.py [hunk 2]
```diff
@@ -83,12 +83,9 @@ def get_segment_class(cls, segment_name) -> Optional[Type["AbstractFlightSegment
...
 
 
-@dataclass
-class AbstractFlightSegment(IFlightPart, ABC):
+class RegisteredSegment(IFlightPart, ABC):
     """
-    Base class for flight path segment.
-
-    As a dataclass, attributes can be set at instantiation.
+    Base class for classes that can be associated with a keyword in missiond definition file.
```
[the remark this answers is not in the document]
Comment (christophe-david) [reply src/fastoad/models/performances/mission/segments/base.py]: Right. Fixed in b824429a53462d4e4ffd0eaef3dc5e9a297181e2

File: src/fastoad/models/performances/mission/segments/tests/test_takeoff_segments.py (hunk 1 above)
Comment (christophe-david) [reply src/fastoad/models/performances/mission/segments/tests/test_takeoff_segments.py:122]: Good question. It just has been lost in the process 😅 
Fixed in b824429a53462d4e4ffd0eaef3dc5e9a297181e2.

**Pull request merged**